src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/columnDefinitions/sc2PublicId.tsx

```diff
@@ -13,7 +13,7 @@ export const sc2PublicIdColumn: ColumnDef<Sample, any> = {
       tooltipStrings={{
         boldText: "Public ID",
         regularText:
-          "Accession ID from public repository or public ID generated by CZ Gen Epi",
```

Comment:
GenBank has 2 IDs, `Isolate name` and `Accession` so specify which one we needs here.